phpBB/docs/events.md

```diff
@@ -1114,6 +1114,76 @@ mcp_warn_user_add_warning_field_before
 * Since: 3.1.0-RC4
 * Purpose: Add content during warning a user - before add warning field.
 
+memberlist_body_page_header_after
+===
+* Locations:
+    + styles/prosilver/template/memberlist_body.html
+* Since: 3.2.6-RC1
+* Purpose: Add data after the page header.
+
+memberlist_body_page_title_before
+===
+* Locations:
+    + styles/prosilver/template/memberlist_body.html
+* Since: 3.2.6-RC1
+* Purpose: Add data before the page title.
+
+memberlist_body_group_name_before
```

Comment:
If sorting alphabetically, memberlist_body_group_name_before should definitely go before memberlist_body_page_title_before.

Reply:
The rest is also not sorted alphabetically. Please try to always sort newly added events alphabetically.